OWML.ModHelper.Menus/ModUIStyleApplier.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModUIStyleApplier : UIStyleApplier
+    {
+        public ModUIStyleApplier():base()
+        {
+            ClearAllArrays();
+        }
+
+        public void Initialize(bool isButton = false, bool isSecondary = false)
```

Comment:
Not used, can it be removed?

Reply:
why do you want to remove all that potentionally useful functionality

Reply:
@Raicuparta same as above

Reply:
Potentially useful in what sense, for future OWML features or something to be used by modders?

If it's just for use within OWML, then definitely remove it.

If it's stuff that's supposed to be useful for modders, please explain the use cases. Whenever we add new functionality to be used by mods, we should add that examples of usage to the available example mods in OWML (the asset load mod is good for that). This isn't something we did with recent new features added, but we should definitely start doing it.

Reply:
for the background thing I nearly used it ones, then realized that making the field simply displays combination noticeably react to hover might make things confusing. It does work though
I didn't really test how ModUIStyleApplier reacts to changing item type, but it simply exposes UIStyleApplier's functionality without reflection

Reply:
for the background thing I nearly used it , then realized that making the field simply displays combination noticeably react to hover might make things confusing. It does work though
I didn't really test how ModUIStyleApplier reacts to changing item type, but it simply exposes UIStyleApplier's functionality without reflection

Reply:
Do you want me to only keep SetAs* in ModUIStyleApplier?

Reply:
TBF that Initialize thing was kinda confusing